### ISSUE TITLE
Discovered Work

### ISSUE THREAD

**Issue by Hamilton-dfp** [Hamilton-dfp/pe4dev#8]
Less than an hour and in the same repo - just do it

In another repo - write up a new story

Large amount of work that blocks current work - take it to PO and team

**Comment by Hamilton-dfp** [comment]
1. How long will this new work take?
2. Will this require an additional PR to be made in another repo?
3. What is the outcome of this new work?
4. Is the outcome of this new work required to meet the sprint goals? The RP Goals?
5. Does the outcome of this new work provide tech enablement?
6. Can this work be done in parallel?
7. Does this work need tracked for auditing or reporting purposes?
